### PR TITLE
Invert chat with a toggle

### LINKED ISSUE #365: [Feature]: Option to switch positions of YTC and filtered TLs
### Checklist

- [X] This is a suggestion for LiveTL, not Hyperchat. I will go to [the Hyperchat repo](https://github.com/LiveTL/HyperChat) to suggest a feature for Hyperchat.
- [X] This is a suggestion for the LiveTL web extension, not LiveTL iOS or LiveTL Android. I will go to the [iOS](https://github.com/LiveTL/ios) or [Android](https://github.com/LiveTL/android) repo to suggest features for those platforms.
- [X] I have checked for a duplicate or similar feature request/issue. I made sure to check closed issues as well.
- [X] I am not reporting a bug. If I am, I will use the bug report issue template.
- [X] By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/LiveTL/LiveTL/blob/develop/CODE_OF_CONDUCT.md)

### Is your feature request related to a problem? Please describe.

Some people might prefer having filtered TLs on top and YTC at the bottom.

### Describe the solution you'd like

Add an option to switch the position of YTC and filtered TLs. Should work for both horizontal and vertical chat split.

### Describe alternatives you've considered

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by KentoNishi**
I messed up #375 so here it is sorry @Eroxl XD

Resolves #365

### CODE REVIEW

**Comment by KentoNishi** [comment]
image

File: src/components/common/RadioGroupStore.svelte [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
   /** Radio button group items. */
   export let items: RadioItem[] = [];
   /** Map to generate items with. Will overwrite `items` prop. */
-  export let map: Map<string, string> | null = null;
+  export let map: Map<any, string> | null = null;
   /** Vertical variant. */
   export let vertical = false;
 
-  const mapToRadioItem = (map: Map<string, string>) => {
+  const mapToRadioItem = (map: Map<any, string>) => {
```
Comment (ChrRubin) [inline src/components/common/RadioGroupStore.svelte]: Can we have these as `string | boolean` instead?

**Review by ChrRubin — CHANGES_REQUESTED** [review]

File: src/components/common/RadioGroupStore.svelte (hunk 1 above)
Comment (KentoNishi) [reply src/components/common/RadioGroupStore.svelte]: sure

**Review by ChrRubin — APPROVED** [review]

**Comment by KentoNishi** [comment]
@r2dev2 approved this irl

**Pull request merged**